src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -68,32 +68,30 @@ public ConsoleLoggerProvider(IOptionsMonitor<ConsoleLoggerOptions> options, IEnu
         [UnsupportedOSPlatformGuard("windows")]
         private static bool DoesConsoleSupportAnsi()
         {
-            string? envVar = Environment.GetEnvironmentVariable("DOTNET_SYSTEM_CONSOLE_ALLOW_ANSI_COLOR_REDIRECTION");
-            if (envVar is not null && (envVar == "1" || envVar.Equals("true", StringComparison.OrdinalIgnoreCase)))
```

Comment:
I am wondering what should be the behavior if someone set `DOTNET_SYSTEM_CONSOLE_ALLOW_ANSI_COLOR_REDIRECTION=1` and the output is not redirected? This env variable will be fully ignored at that time. is this the expected behavior?

Reply:
CC @hach-que